examples/graphene_apps/python_workloads/fibonacci/graphene_sgx/build_gsc_fibonacci_workload.sh

```diff
@@ -38,29 +39,13 @@ if [ "$GSC_IMAGE_EXISTS" = "yes" ]; then
 fi
 
 # Manifest files
-MANIFEST_FILE_DIR="${TCF_HOME}/tc/graphene/python_worker/graphene_sgx/manifest"
-MANIFEST_FILES="python.manifest
-sh.manifest
-gcc.manifest
-collect2.manifest
-ld.manifest"
-# Generate list of manifest files
-LIST_MANIFEST_FILES=""
-for f in $MANIFEST_FILES
-do
-  FILE_NAME=${MANIFEST_FILE_DIR}/$f
-  if [ ! -f $FILE_NAME ]; then
-    echo "ERROR:Manifest file $FILE_NAME doesn't exist"
-    exit
-  fi
-  LIST_MANIFEST_FILES+=${MANIFEST_FILE_DIR}/$f
-  LIST_MANIFEST_FILES+=" "
-done
-echo $LIST_MANIFEST_FILES
+
+MANIFEST_FILE="./avalon.manifest" #"${TCF_HOME}tc/graphene/python_worker/graphene_sgx/manifest/python.manifest"
```

Comment:
its better to address file with absolute path than relative path, it leads to bugs if run from different paths

Reply:
Addressed the file with absolute path.